qutebrowser/completion/models/miscmodels.py

```diff
@@ -111,6 +111,27 @@ def __init__(self, match_field='url', parent=None):
                 match_field))
 
 
+class BookmarkCompletionModel(base.BaseCompletionModel):
```

Comment:
Maybe this could be merged into `QuickmarkCompletionModel` as well, with a parameter (similar to `match_field` which says which object to use to get the marks (i.e. `bookmark-manager` or `quickmark-manager`)?